### PR TITLE
Add gg.tesseract.Tesseract

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** http://tesseract.gg/forum/viewtopic.php?pid=3268#p3268
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build gg.tesseract.Tesseract

**Review by hfiguiere — COMMENTED** [review]
Did you submit the desktop and appstream files upsstream?

File: gg.tesseract.Tesseract.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - mkdir -p ${FLATPAK_DEST}/lib/tesseract
+      - cp -a ../media/ ../config/ ../bin_unix/ ${FLATPAK_DEST}/lib/tesseract
+      - install -Dm755 ../tesseract_unix ${FLATPAK_DEST}/bin/tesseract
+      - install -Dm644 ../media/interface/cube.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 ../gg.tesseract.Tesseract.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ../gg.tesseract.Tesseract.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: svn
+        url: svn://svn.tuxfamily.org/svnroot/tesseract/main
+        commit: 2549
```
Comment (hfiguiere) [inline gg.tesseract.Tesseract.yaml:29]:
```
(flatpak-builder:2): flatpak-builder-WARNING **: 19:29:38.806: Unknown property commit for type BuilderSourceSvn
```

So it's not pinned to any revision.

**Comment by Mailaender** [comment]
I am trying to contact the developer as I don't know how to contribute with WebSVN.

**Comment by Mailaender** [comment]
bot, build gg.tesseract.Tesseract

**Comment by hfiguiere** [comment]
> I am trying to contact the developer as I don't know how to contribute with WebSVN.

you can't. usually one send patches through the bug tracker.

**Pull request closed without merging**